### PR TITLE
Release 27

### PR DESCRIPTION
**Pull request by blackboxsw**
Bug fix and bump upstream version of impish to 27.0.2 (since we already queued 27.0.1 in a -proposed upload)

git cherry-pick d286d8f5185af0422118ee369392b3e101fe53ba..b544ab19b3d1a49b24e5c18f85aad4b0b5c3da
Plus sort minor cherry-pick conflicts.

We are limiting this fix to just the changeset in https://github.com/canonical/ubuntu-advantage-client/pull/1572 to avoid introducing more features/funcationality that adds SRU additional risk.

Additionall I added a 27.0.1 changelog entries since we already queued a release into impish-proposed for  for packaging .gitignore packaging fixes.

## Test Steps
edit debian/changelog for the release you are targetting and build it
```bash

# setup a schroot for focal riscv64
sudo sbuild-launchpad-chroot create --architecture="riscv64" "--name=focal-riscv64" --series=focal
s/UNRELEASED/focal/ debian/changelog
git commit -m 'build wip' debian/changelog
build-package 
sbuild-it ../out/*dsc riscv64
```
## Desired commit type::

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,22 @@
+ubuntu-advantage-tools (27.0.2) trusty; urgency=medium
```
Comment (orndorffgrant) [inline debian/changelog:1]: A build commit slipped in here
```suggestion
ubuntu-advantage-tools (27.0.2) impish; urgency=medium
```

**Review by orndorffgrant — APPROVED** [review]
Thanks a ton for cherry-picking this all over and working through the details! All of my manual builds have worked out as expected with correct go versions. Simulating a scenario where go doesn't get installed as a build-dep also worked out. And double checked the versions of go each release focal or greater supports riscv.

Approving, pending a successful jenkins build.

Also looks like a tiny changelog trusty commit slipped in that should be removed

File: debian/changelog (hunk 1 above)
Comment (blackboxsw) [reply debian/changelog:1]: +1 dropping pending successful jenkinsrun.

**Comment by blackboxsw** [comment]
Expected CI failure on release-27 branch as we lack a fix from main
```bash
      Expected: a string matching 'USN-4539-1: AWL vulnerability
      Found CVEs:
      https://ubuntu.com/security/CVE-2020-11728
      1 affected package is installed: awl
      \(1/1\) awl:
      Ubuntu security engineers are investigating this issue.
      1 package is still affected: awl
      .*✘.* USN-4539-1 is not resolved.'
           but: was 'USN-4539-1: AWL vulnerability\nFound CVEs:\nhttps://ubuntu.com/security/CVE-2020-11728\n1 affected package is installed: awl\n(1/1) awl:\nSorry, no fix is available.\n1 package is still affected: awl\n\x1b[91m✘\x1b[0m USN-4539-1 is not resolved.'
```

**Pull request merged**